packages/wp-source/src/libraries/__tests__/__snapshots__/populate.tests.ts.snap

```diff
@@ -61,7 +61,10 @@ Object {
         ],
         "wp:term": Array [
           Array [
-            1,
+            Object {
+              "id": 1,
+              "schema": "taxonomyEntity",
+            },
```

Comment:
`normalizr` replaces IDs with an object with `id` and `schema`, now that term entities can have different schemas, so I had to update a lot of snapshots. Most of the changes are like this one, in the property `_embedded["wp:term"]` of post type entities.